integration/docker/inspect_test.go

```diff
@@ -23,11 +23,11 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-func inspectFormatOptions (formatOption string) TableEntry {
+func inspectFormatOptions(formatOption string) TableEntry {
 	return Entry(fmt.Sprintf("inspect with %s", formatOption), formatOption)
 }
 
-var _= Describe ("inspect", func() {
+var _ = Describe ("inspect", func() {
```

Comment:
Hi. I think you only fixed one ' ' anomaly. There was another reported with the `Describe (` as well ;-) (two space errors on the same line that is). `go-static-checks` is still complaining for me. Did you check your fixes by running it?